packages/babel-node/src/_babel-node.ts

```diff
@@ -16,6 +16,7 @@ import type { PluginAPI, PluginObject } from "@babel/core";
 
 const require = createRequire(import.meta.url);
 
+// @ts-expect-error TS2339: Property 'default' does not exist on type 'typeof import("./node_modules/commander-BABEL_8_BREAKING-true/typings/index")'
 const program = commander.default.program as commander.Command;
```

Comment:
Can we maybe add this to `lib/third-party-libs.d.ts`?
```ts
declare module "commander" {
  import type { Command } from "commander-BABEL_8_BREAKING-true";
  const _default: { program: Command };
  export { _default as default };
}
```

(and remove it from tsconfig's paths)